src/Geta.404Handler/Controllers/NotFoundRedirectController.cs

```diff
@@ -86,7 +87,7 @@ public ActionResult Index(int? pageNumber, string searchWord, int? pageSize, boo
         public ActionResult SaveSuggestion(string oldUrl, string newUrl, string skipWildCardAppend, int? pageNumber, int? pageSize)
         {
             CheckAccess();
-            SaveRedirect(oldUrl, newUrl, skipWildCardAppend);
+            SaveRedirect(oldUrl, newUrl, skipWildCardAppend, (int)RedirectType.Permanent);
```

Comment:
Instead of casting to `int`, make the parameter of `RedirectType`.

Reply:
I've made the change to use the enum type